objectstore/struct.go

```diff
@@ -0,0 +1,34 @@
+package objectstore
+
+const (
+	Enable        = "enable"
+	VolumeName    = "name"
+	ObjectStoreID = "id"
+)
+
+// Objectstoreinfo returns current objectstore details
+// swagger:model
+type ObjectstoreInfo struct {
```

Comment:
Shouldn't this be in `api.proto`?

Reply:
missed this in offline meeting discussion. I have added this into api.proto